docs/getting-started-guide/faqs.rst

```diff
@@ -26,6 +24,9 @@ How does ``xcdat`` interpret dataset metadata?
 ``ds.temporal.average("ts")``). This argument is internally mapped to ``cf_xarray``
 mapping tables that interpret the CF attributes.
 
+xCDAT also includes its own "fall-back" mapping table that maps axes to their commonly accepted names
+(e.g., X axis maps to "longitude" and "lon").
```

Comment:
I should create a table for the "fall-back" map that xCDAT uses here in the docs.